website/pages-parts/LandingPage/Testimonials.tsx

```diff
@@ -82,46 +223,52 @@ export function Testimonials() {
       </h3>
       <Heading>Testimonials</Heading>
       <div className={styles.testimonials}>
-        <Testimonial
-          company="Resolve"
-          description={
-            <>
-              uniforms is the backbone of our data-intensive web-applications.
-              We have about 200 different forms, from very simple ones, to ones
-              that are filled with complex data-loading conditional form
-              components, which create an incredible UX for our users. And if
-              you really need to push the limits of what you can do with forms,
-              I would highly recommend{' '}
-              <b>
-                <a href="https://www.vazco.eu/" target="blank">
-                  reaching out to Vazco
-                </a>
-              </b>{' '}
-              themselves for expert advice.
-            </>
-          }
-          position="CTO and Co-Founder"
-          who="Florian Bienefelt"
-          avatar="img/avatar/florian-bienefelt.jpg"
-          linkGithub="https://github.com/Floriferous"
-          linkLinkedin="https://ch.linkedin.com/in/florianbienefelt"
-        />
-        <Testimonial
-          company="Toptal"
-          description={
-            <>
-              uniforms is my go-to solution for quite a while. Great holistic
-              approach to tackle forms. I especially love the approach to making
-              custom form layouts. Developer experience par-excellence.
-            </>
-          }
-          mirror
-          position="Front-end Platform Architect"
-          who="Viktor Bezdek"
-          avatar="img/avatar/viktor-bezdek.jpg"
-          linkGithub="https://github.com/viktorbezdek"
-          linkLinkedin="https://www.linkedin.com/in/viktorbezdek/"
-        />
+        <button
+          className={classNames(
+            styles['arrow-button'],
+            styles['left-arrow-button'],
+          )}
+          onClick={setPrevSlide}
+        >
+          <img
+            alt="arrow-left"
+            src="assets/arrow-left.svg"
+            className={styles['arrow']}
+          />
+        </button>
+
+        <div className={styles['carousel-container']}>
+          <div
+            className={styles['carousel']}
+            style={{
+              transform: `translate(-${slide * 50}%)`,
+            }}
+          >
+            {testimonials.map((testimonial, testimonialIdx) => {
+              return (
+                <Testimonial
+                  key={testimonialIdx}
+                  {...testimonial}
+                  mirror={testimonialIdx !== slide}
+                />
+              );
```

Comment:
1. (optional) This could be simplified not to use the `return` at all.
2. (optional) Maybe passing the `testimonial` as an object would be better than unwrapping it (fewer types).